kloppy/infra/serializers/event/statsbomb/deserializer.py

```diff
@@ -820,6 +896,21 @@ def deserialize(self, inputs: StatsBombInputs) -> EventDataset:
                     )
                     new_events.append(generic_event)
 
+                # Add possible aerial won - Applicable to multiple event types
+                for type_name in ["shot", "clearance", "miscontrol", "pass"]:
+                    if (
+                        type_name in raw_event
+                        and "aerial_won" in raw_event[type_name]
+                    ):
+                        duel_event_kwargs = _parse_aerial_won_duel(
+                            raw_event=raw_event, type_name=type_name
+                        )
+                        duel_event = self.event_factory.build_duel(
+                            **duel_event_kwargs,
+                            **generic_event_kwargs,
+                        )
+                        new_events.append(duel_event)
```

Comment:
Should this happen before the shot/clearance/miscontrol/pass event or after (using `.append`)?